cloudinit/config/cc_install_hotplug.py

```diff
@@ -21,7 +21,7 @@
         This module will install the udev rules to enable hotplug if
         supported by the datasource and enabled in the userdata. The udev
         rules will be installed as
-        ``/etc/udev/rules.d/10-cloud-init-hook-hotplug.rules``.
+        ``/etc/udev/rules.d/99-cloud-init-hook-hotplug.rules``.
```

Comment:
Yes, we will need to rename that file at `preinst upgrade` time in the downstream ubuntu branches.

This is needed because `ENV{ID_NET_DRIVER}` is not defined when the rule is tested when defined in `10-cloud-init-hook-hotplug.rules`, so the rule does not match. I tried also with `DRIVERS` but the rule didn't match at NIC removal time.

Reply:
PR for noble: https://github.com/canonical/cloud-init/pull/4851, I will create similar ones for f, j, m.